codespell_lib/data/dictionary.txt

```diff
@@ -10197,12 +10222,15 @@ delclaration->declaration
 delection->detection, deletion, selection,
 delections->detections, deletions, selections,
 delele->delete
+delelete->delete
+deleleted->deleted
```

Comment:
You may add `deleletes` and `deleleting` if you can, otherwise I'll add them later myself.